model-optimizer/extensions/front/tf/while_ext.py

```diff
@@ -0,0 +1,207 @@
+"""
+ Copyright (C) 2017-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import copy
+
+from extensions.front.onnx.loop_ext import connect_body_input, connect_body_output
```

Comment:
That makes impossible to use MO for TensorFlow without ONNX.

Reply:
Thanks for creating a ticket! Will be fixed soon.